src/components/Demo1/DemoOneBackground.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+import backgroundDemoOne from '../../assets/images/background-demo-one.png'
+
+export const DemoOneBackground = styled.div(() => ({
```

Comment:
Why all files got prefixes of `DemoOne` when it's in a folder `Demo1`? 

Can we call just `Background` and 'Header' and so on?